giskard_vision/landmark_detection/detectors/cropping_detector.py

```diff
@@ -0,0 +1,28 @@
+from ..dataloaders.wrappers import CroppedDataLoader
+from ..marks.facial_parts import FacialParts
+from .base import LandmarkDetectionBaseDetector
+
+try:
+    from giskard.scanner.decorators import detector
+except (ImportError, ModuleNotFoundError) as e:
+    e.msg = "Please install giskard to use custom detectors"
+    raise e
```

Comment:
As we previously discussed, please try to create a `maybe_detector` decorator that gets imported conditionally